<ui version="4.0" >
 <class>LibraryDialog</class>
 <widget class="QWidget" name="LibraryDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>463</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="palette" >
   <palette>
    <active>
     <colorrole role="WindowText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>208</red>
        <green>208</green>
        <blue>208</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>247</red>
        <green>247</green>
        <blue>247</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>104</red>
        <green>104</green>
        <blue>104</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>139</red>
        <green>139</green>
        <blue>139</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>128</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="HighlightedText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Link" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="LinkVisited" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>0</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>231</red>
        <green>231</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>208</red>
        <green>208</green>
        <blue>208</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>247</red>
        <green>247</green>
        <blue>247</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>104</red>
        <green>104</green>
        <blue>104</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>139</red>
        <green>139</green>
        <blue>139</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>192</red>
        <green>192</green>
        <blue>192</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="HighlightedText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Link" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="LinkVisited" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>0</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>231</red>
        <green>231</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>104</red>
        <green>104</green>
        <blue>104</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>208</red>
        <green>208</green>
        <blue>208</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>247</red>
        <green>247</green>
        <blue>247</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>104</red>
        <green>104</green>
        <blue>104</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>139</red>
        <green>139</green>
        <blue>139</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>104</red>
        <green>104</green>
        <blue>104</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>104</red>
        <green>104</green>
        <blue>104</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>128</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="HighlightedText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Link" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="LinkVisited" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>0</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>231</red>
        <green>231</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="contextMenuPolicy" >
   <enum>Qt::NoContextMenu</enum>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox_5" >
     <property name="title" >
      <string>Library</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="1" colspan="2" >
       <widget class="QLabel" name="label_3" >
        <property name="styleSheet" >
         <string>background-color: rgb(253, 252, 248);</string>
        </property>
        <property name="text" >
         <string>Exploring My RetroShare Library</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QPushButton" name="tileView_btn_library" >
        <property name="text" >
         <string>Tile view</string>
        </property>
        <property name="flat" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="4" >
       <widget class="QPushButton" name="showDetails_btn_library" >
        <property name="text" >
         <string>Show Details</string>
        </property>
        <property name="flat" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="4" >
       <widget class="QListView" name="organizerListView" >
        <property name="contextMenuPolicy" >
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <property name="movement" >
         <enum>QListView::Free</enum>
        </property>
        <property name="viewMode" >
         <enum>QListView::IconMode</enum>
        </property>
        <property name="selectionRectVisible" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QPushButton" name="createAlbum_btn_library" >
        <property name="styleSheet" >
         <string/>
        </property>
        <property name="text" >
         <string>Create Album</string>
        </property>
        <property name="flat" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QPushButton" name="deleteAlbum_btn_library" >
        <property name="styleSheet" >
         <string/>
        </property>
        <property name="text" >
         <string>Delete Album</string>
        </property>
        <property name="flat" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="3" >
       <widget class="QLineEdit" name="findEditmain" />
      </item>
      <item row="2" column="4" >
       <widget class="QPushButton" name="find_btn_library" >
        <property name="styleSheet" >
         <string>pushButton_41:hover{ color : white;}</string>
        </property>
        <property name="text" >
         <string>Find</string>
        </property>
        <property name="flat" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item rowspan="3" row="0" column="0" >
       <widget class="QTabWidget" name="tabWidget_2" >
        <property name="tabShape" >
         <enum>QTabWidget::Triangular</enum>
        </property>
        <property name="currentIndex" >
         <number>1</number>
        </property>
        <widget class="QWidget" name="tab_13" >
         <attribute name="title" >
          <string>Folder</string>
         </attribute>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QComboBox" name="comboBox" >
            <property name="styleSheet" >
             <string>border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
            </property>
            <item>
             <property name="text" >
              <string>All File Types</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Application</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Archive</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_14" >
         <attribute name="title" >
          <string>Organizer</string>
         </attribute>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QTreeView" name="organizertreeView" >
            <property name="font" >
             <font>
              <family>Utopia</family>
              <pointsize>8</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QPushButton" name="shareFiles_btn" >
            <property name="styleSheet" >
             <string>border-color: rgb(0, 0, 0);</string>
            </property>
            <property name="text" >
             <string>Share Files..</string>
            </property>
            <property name="flat" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>organizertreeView</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>organizerListView</receiver>
   <slot>setRootIndex(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>139</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel" >
     <x>439</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
